src/main/java/nextstep/courses/domain/session/Session.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.courses.domain.session;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import nextstep.courses.domain.charge.Charge;
+import nextstep.courses.domain.enrollment.Enrollment;
+import nextstep.courses.exceptions.AlreadyEnrollmentException;
+import nextstep.users.domain.NsUser;
+
+import java.util.List;
+
+@Getter
+@Builder
+@AllArgsConstructor
+public class Session {
+
+    private Sessioninfo sessioninfo;
+
+    private Charge charge;
+
+    private final SessionPeriod sessionPeriod;
+
+    private final Enrollment enrollment;
+
+    public void enroll(NsUser nsUser, List<NsUser> students) throws AlreadyEnrollmentException {
```

Comment:
```suggestion
    public void enroll(NsUser nsUser) throws AlreadyEnrollmentException {
```

도메인 객체 구현에서는 굳이 학생 목록을 List로 전달하지 않아도 되지 않을까?